config/puma.rb

```diff
@@ -33,6 +33,9 @@
 # Allow puma to be restarted by `bin/rails restart` command.
 plugin :tmp_restart
 
+# Run the Solid Queue supervisor inside of Puma for single-server deployments
+plugin :solid_queue if ENV["SOLID_QUEUE_IN_PUMA"]
+
```

Comment:
There has been back and forth discussion in SolidQueue on whether it should run as a puma pluging or separatly. 

At first I was using the puma pluging but then as it was about to be removed (finally not) I removed it. now there is a dedicated process in Kamal for the job https://github.com/adrienpoly/rubyvideo/blob/b13149b5177486b2bd37eb049db8e0e4802ffc70/config/deploy.yml#L12-L15

and in  procfile.dev we also start solid queue

Reply:
Isn't this only running the plugin when the ENV variable is present?

Reply:
yes ok let's keep it like that